tools/sklearn/test-data/feature_selection_result12

```diff
@@ -1,11 +1,11 @@
 0	1
-143.762620712	-1.1796457192799998
--88.5787166225	-2.5710918402200003
--82.8452345578	-0.168636324107
-72.4951388149	0.991068834926
-11.805182128	-0.7096855607860001
--63.9354970901	0.9841122108220001
-126.32584079600001	0.35353444883900004
-23.0341392692	1.03188231893
-67.6714937696	-0.8214378651719999
-47.39275848810001	-0.0942409319417
+143.762620712	-0.330941870584
+-88.5787166225	1.08055532812
+-82.8452345578	0.272541389247
+72.4951388149	-0.26868660527800003
+11.805182128	1.0360467096600001
+-63.9354970901	-0.101485840571
+126.32584079600001	-0.35999834017899995
+23.0341392692	0.5185404651359999
+67.6714937696	-0.115688051547
+47.39275848810001	-0.7850965413680001
```

Comment:
It seems the a different column is selected. Do the two columns have the same importance? or other reason?